### DISCUSSION TITLE
Adding custom control to the search bar

### DISCUSSION THREAD

**Discussion by H-Lo** [vitmalina/w2ui#2395] [General]
The idea is to add a dropdown into search bar, next to search input control and buttons "edit", "delete", "add new".
That dropdown would hold statuses of my items so my users can easily filter by its value.
For example, if user selects "DONE" from a dropdown, I'd filter all items to have its status == "DONE".
If users selects "Show All",  no filtering would be done.

Is that possible?

P.S. I am aware that control could be outside Grid but it would be more user friendly if it could be placed in search bar.

**Comment by vitmalina** [comment]
You can already do this. The grids toolbar is regular w2toolbar where you can add new buttons. You can also insert custom items before predefined ones. See https://w2ui.com/web/docs/2.0/w2grid.toolbar
